### PR TITLE
chore(query-bar): pass conditions into styleguide

### PR DESCRIPTION
**Pull request by ChenCodes**
Changes in this PR:
- Pass conditions into styleguide for Query Bar
- Default size to be empty string for Avatar.js
- Resolves two flow errors currently being shown in the console

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ChenCodes has signed the CLA. Thanks for the pull request!

File: src/components/avatar/Avatar.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ class Avatar extends React.PureComponent<Props, State> {
     };
 
     render() {
-        const { avatarUrl, className, name, id, size }: Props = this.props;
+        const { avatarUrl, className, name, id, size = '' }: Props = this.props;
```
Comment (ChenCodes) [inline src/components/avatar/Avatar.js]: I included this change as it helps clean up a flow error that got through the flow check, didn't think we would need a separate ticket + new pr just for a one liner change

**Pull request merged**